javascript/javascript1/week1/myscripts.js

```diff
@@ -0,0 +1,81 @@
+// Task 1
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Task 2
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture - dogYearOfBirth + 60;
+let shouldShowResultInDogYear = true;
+
+if (shouldShowResultInDogYear == true) {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+} else {
+  let humanYear = dogYear - 60;
+  console.log(
+    "Your dog will be " + humanYear + " human years old in " + dogYearFuture
+  );
+};
+
+// Task 3
+// Peter house prise calculation
+let peter = { wide: 8, deep: 10, high: 10, gardenSizeInM2: 100 };
+let peterVolumeInMeters = peter.wide * peter.deep * peter.high;
+let peterHousePrise =
+  peterVolumeInMeters * 2.5 * 1000 + peter.gardenSizeInM2 * 300;
+if (peterHousePrise > 2500000) {
+  console.log("Peter is paying too much");
+} else {
```

Comment:
what if `peterHousePrice === 2500000`?

Reply:
Its will come wrong result. for that condition, I have to use peterHousePrise >=